build-in-container/scripts/build-mariner.sh

```diff
@@ -6,10 +6,32 @@
 set -e
 set -x
 
+# parse args passed to container
+args=("$@")
+for ((i = 0; i < ${#args[@]}; i=i+2)); do
```

Comment:
For more complex arg parssing, I really like this design:
https://github.com/microsoft/CBL-Mariner/blob/9b8eca45679803c9c2473ccd81ae37ccdf2fb7f8/SPECS/xorg-x11-server/xvfb-run.sh#L115

Or slightly more verbose: https://github.com/microsoft/CBL-Mariner/blob/9b8eca45679803c9c2473ccd81ae37ccdf2fb7f8/SPECS/etcd/generate_source_tarball.sh#L21